plugins/main/bridge/bridge.go

```diff
@@ -174,9 +175,9 @@ func collectVlanTrunk(vlanTrunk []*VlanTrunk) ([]int, error) {
 			for v := minID; v <= maxID; v++ {
 				vlanMap[v] = struct{}{}
 			}
-		} else if item.MinID == nil && item.MaxID != nil {
+		case item.MinID == nil && item.MaxID != nil:
```

Comment:
nit: maybe omit the != nil ? you know one of them is empty ; just checking for the empty ones would be OK, right ?

Reply:
I will consider to add detail condition to make sure others could read it and understand it immediately.

Reply:
the 2 of them could be empty, the != nil is needed I think

Reply:
yep, because it's switch-case
i'm not so sure about that.

Reply:
we would need more unit tests then, to assert this works as intended :)

Reply:
I take a look in test.go
but I'm not sure how to add a unit test without in cmdAddDel Test case hmm.

Reply:
all test is via cmdAddDelTest.
So it's not easy to insert an unit test for collectVlanTrunk.

Reply:
I would just add a new context in the describe block and have something like:

```golang

DescribeTable(
    "collectVlanTrunk succeeds", 
    func(vlanTrunks ...*vlanTrunk, expectedVIDs []int) {
        Expect(vlancollectVlanTrunk(vlanTrunks)).To(Equal(expectedVIDs))
    }, 
    Entry("", &vlanTrunk{}, []int{}),
    Entry("", &vlanTrunk{minID: 100, maxID: 105}, []int{100, 101, 102, 103, 104, 105}),
...
)

DescribeTable(
    "collectVlanTrunk **fails**", 
    func(vlanTrunks ...*vlanTrunk, expectedError error) {
        Expect(vlancollectVlanTrunk(vlanTrunks)).To(MatchError(expectedError))
    }, 
    Entry("", &vlanTrunk{minID: 100}, fmt.Errorf("minID and maxID should be configured simultaneously, maxID is missing")),
    Entry("", &vlanTrunk{maxID: 100}, "minID and maxID should be configured simultaneously, minID is missing"),
    Entry("", &vlanTrunk{minID: -100, maxID: 105}, "incorrect trunk minID parameter"),
    Entry("", &vlanTrunk{minID: 10000, maxID: 105}, "incorrect trunk minID parameter"),
    Entry("", &vlanTrunk{minID: 100, maxID: 10000}, "incorrect trunk minID parameter"),
    Entry("", &vlanTrunk{minID: 100, maxID: -1}, "incorrect trunk minID parameter"),
...
...
)

```

Reply:
Unit tests are included in latest commit